openff/interchange/interop/openmm/_nonbonded.py

```diff
@@ -650,28 +672,27 @@ def _create_multiple_nonbonded_forces(
     coul_14_force.addPerBondParameter("qq")
     coul_14_force.setUsesPeriodicBoundaryConditions(interchange.box is not None)
 
-    bonds = [
-        sorted(interchange.topology.atom_index(a) for a in bond.atoms)
-        for bond in interchange.topology.bonds
-    ]
-
     coul_14, vdw_14 = _get_14_scaling_factors(data)
 
-    electrostatics_force.createExceptionsFromBonds(
-        bonds=bonds,
-        coulomb14Scale=coul_14,
-        lj14Scale=vdw_14,
-    )
+    openmm_pairs = list()
 
-    for i in range(electrostatics_force.getNumExceptions()):
-        (p1, p2, q, sig, eps) = electrostatics_force.getExceptionParameters(i)
+    for atom1, atom2 in _get_14_pairs(interchange.topology):
+        openff_indices = (
+            interchange.topology.atom_index(atom1),
+            interchange.topology.atom_index(atom2),
+        )
 
-        # If the interactions are both zero, assume this is a 1-2 or 1-3 interaction
-        if q._value == 0 and eps._value == 0:
-            pass
-        else:
-            # Assume this is a 1-4 interaction
```

Comment:
This was fun to replace - this assumption fails when there's no vdW interactions!